### PR TITLE
docs: fix json

### PR DESCRIPTION
**Pull request by MrMartinR**
This PR fixes the issue https://github.com/parse-community/docs/issues/884

### CODE REVIEW

**Comment by MrMartinR** [comment]
Hi guys, I fixed this small json example, @parse-community/docs-review

File: _includes/rest/hooks.md [hunk 1]
```diff
@@ -220,7 +220,7 @@ and perform a `parse deploy` the usual way.
 To create a new function webhook post to <code class="highlighter-rouge"><span class="custom-parse-server-mount">/parse/</span>hooks/functions</code> with payload in the format
 
 ```jsonc
-{"functionName" : x, "url" : y}
+{"functionName": "foo", "url": "https://api.example.com/foo"}
```
Comment (mtrezza) [inline _includes/rest/hooks.md]:
```suggestion
{
  "functionName": "foo",
  "url": "https://api.example.com/foo"
}
```

**Review by mtrezza — COMMENTED** [review]
Shouldn't this also be prettified?

File: _includes/rest/hooks.md (hunk 1 above)
Comment (MrMartinR) [reply _includes/rest/hooks.md]: > Shouldn't this also be prettified?
Done!

**Review by dblythy — APPROVED** [review]
LGTM

**Review by mtrezza — APPROVED** [review]

**Pull request merged**